packages/common/src/core/types/IHubLocation.ts

```diff
@@ -26,3 +27,20 @@ export interface IHubLocation {
   graphic?: any;
   geoJson?: any;
 }
+
+/**
+ * Location options for the location picker
+ */
+export interface IHubLocationOption {
```

Comment:
same here - if we're hoisting this interface here, can we remove it from `opendata-ui` so we don't end up with two out-of-sync instances?